tests/integration/api_v2/subscribeApi/feesEvent.test.js

```diff
@@ -0,0 +1,38 @@
+/*
```

Comment:
`tests/integration/api_v2/subscribeApi/feesEvent.test.js` to `tests/integration/api_v2/events/fees.test.js`